odoo_tools/cli/addon.py

```diff
@@ -26,25 +26,38 @@ def cli():
 
 @cli.command()
 @click.argument("name")
-@click.option("-v", "--version", "version")
-@click.option("-p", "--pr", "pr")
-@click.option("-o", "--odoo", "odoo", default=True)
-@click.option("--upgrade", "upgrade", is_flag=True, default=False)
+@click.option(
+    "-v", "--version", "version", help="the version of the addon you want to install"
+)
+@click.option("-p", "--pr", "pr", help="the URL of the pull request you want to add")
+@click.option(
+    "--odoo/--no-odoo",
+    "odoo",
+    default=True,
+    help="use --no-odoo to install a python module which is not an Odoo addon",
+)
+@click.option(
+    "--upgrade",
+    "upgrade",
+    is_flag=True,
+    default=False,
+    help="upgrade will update the addon to the latest available version on PyPI",
+)
 def add(name, version=None, pr=None, odoo=True, upgrade=False):
     """Update the project's requirements for a given package (odoo or not).
 
     * If the module is not already listed in the requirements.txt, check the
       latest version of the module on PyPI and use that version
 
     * Otherwise:
+
         * if the version is the same, do nothing. Otherwise, prompt the user
         * If the addon is present as a PR, prompt the user
 
     """
-    # TODO: get odoo version from project
     # TODO: centralize printing/logging in `ui` utils
     click.secho(f"Adding: {name}", fg="green")
-    pkg = Package(name, odoo=odoo)
+    pkg = Package(name, odoo=odoo, odoo_serie=get_current_version(serie_only=True))
```

Comment:
hmm do we really need it here? The odoo serie will always be the same. 
I'd rather call `get_current_version` in `get_last_pypi_version` if  the `odoo` flag is set.
Do you see any case where you might want to pass it w/o changing the proj version?

Reply:
I'm fine with this. It was a question of dependencies among packages, I was too lazy to check this yesterday. 

I'll update the PR.